src/main/java/Bellroy/Task.java

```diff
@@ -0,0 +1,28 @@
+package Bellroy;
```

Comment:
Would be good if can create a separate package. For example, all tasks-related classes such as Event, Deadline, and Todo can be under Bellroy.Task package.